src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,105 @@
 [
   {
+    "name": "Google Cloud Vision",
+    "documentation": "https://cloud.google.com/vision/docs/reference/libraries",
+    "description":"The Google Cloud Vision API provides vision features including image labeling, face and landmark detection, optical character recognition (OCR), and tagging of explicit content.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/vision.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Vision",
```

Comment:
_Java Client_ seems odd.  _Java Client Libraries for Google Cloud Vision_?  _Google Cloud Vision Client Libraries for Java_?  The libraries pages themselves use _"Cloud Vision API Client Libraries"_.

Reply:
Java Client Libraries for the Google Cloud Vision API